src/stylesheets/_resource_tile.scss

```diff
@@ -3,11 +3,18 @@
     height: 31.2rem;
     padding: 3.6rem 1.8rem 1.8rem;
 
+    // TODO: NO NO NO NO! BAD! WAS IMPOSSIBLE TO FIND!
     .context-menu {
         position: absolute;
         top: 0.6rem;
         right: 0.6rem;
     }
+
+    .resource-view-more-link {
```

Comment:
This could be done without nesting.

Reply:
Could you explain further? I don't see what you mean.

Reply:
This could be changed to something like resource-tile__view-more-link, then It doesn't have to be within resource-tile.